bpf/process/types/basic.h

```diff
@@ -73,6 +73,7 @@ enum {
 	ACTION_COPYFD = 5,
 	ACTION_GETURL = 6,
 	ACTION_DNSLOOKUP = 7,
+	ACTION_NOPOST = 8,
```

Comment:
Hey @olsajiri so we won't get the override event?

Reply:
hum, not sure what you mean, there's OVERRIDE enum earlier in the list